### DISCUSSION TITLE
Custom Filter Error: "Cannot resolve keyword .. into field"

### DISCUSSION THREAD

**Discussion by irvinktang** [carltongibson/django-filter#1423] [Q&A]
Hi there,

I'm trying to create a custom filter right now in my Graphene Django project using django-filter.

I followed the documentation's example of creating a custom filter. My example:
```
class BatchAnalyticsFilter(django_filters.OrderingFilter):
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)

        self.extra["choices"] += [
            ("total_scans", "total_scans"),
            ("-total_scans", "-total_scans")
        ]

    def filter(self, qs, value):
        if any(v in ["total_scans", "-total_scans"] for v in value):
            print(qs)
        return super().filter(qs, value)
```

However, it's telling me that `total_scans` can't be resolved into a field and then lists the fields that I've specified in my actual model. How can I get around this error?

**Comment by carltongibson** [comment]
There’s not sufficient info here to be able to say anything.
